src/client/pages/Checkout/CheckoutDetails/CheckoutDetails.tsx

```diff
@@ -105,6 +106,8 @@ export const CheckoutDetails = () => {
       >
         <PaymentInfo {...priceData} />
       </Footer>
+
+      <CheckoutErrorModal isVisible={error !== undefined} onRetry={onRetry} />
```

Comment:
A bit late to the party here, but if the server returns `error: null` the modal will not show - is that intended? Maybe the server never sends that?

Reply:
Hmmm at least it's typed as `ApolloError | undefined` 🤔